panther_power_control/src/relays_node.py

```diff
@@ -13,25 +12,25 @@
 from panther_msgs.msg import DriverState, IOState
 
 
-@dataclass
-class PatherGPIO:
-    MOTOR_ON = 6  # Pin to enable motor controllers
-    STAGE2_INPUT = 22  # Check if power can be forwarded to motor controllers
-
-    def __setattr__(self, name: str, value: int) -> None:
-        raise AttributeError(f'Can\'t reassign constant {name}')
-
-
 class RelaysNode:
     def __init__(self, name: str) -> None:
-        rospy.init_node(name, anonymous=False)
+        self._node_name = name
+        rospy.init_node(self._node_name, anonymous=False)
 
         self._lock = Lock()
 
-        self._pins = PatherGPIO()
-        self._setup_gpio()
+        self._chip = gpiod.Chip('gpiochip0', gpiod.Chip.OPEN_BY_NAME)
+
+        line_names = {'MOTOR_ON': 6, 'STAGE2_INPUT': 22}
```

Comment:
```suggestion
        line_names = {
            'MOTOR_ON': 6,
            'STAGE2_INPUT': 22,
        }
```
Personally I like this formatting for dicts - with `,` after last element